dist/firmware/firmware.json

```diff
@@ -25,6 +25,29 @@
                 "offset": "0x200000"
             }
         },
+        {
```

Comment:
Is this supposed to be for the older V2 hardware that used the ESP01 modules or something else like an ESP01 carrier in a wemos slot or something?

Reply:
ESP01S has an ESP01 footprint with the larger 4MB flash size. That would allow V4 to run on the V2 HW.

Reply:
Could you change the name to "ESPixelStick V2 ESP01S"? Thanks!